pkg/controller/jobs/job/job_multikueue_adapter.go

```diff
@@ -74,21 +73,19 @@ func (b *multiKueueAdapter) SyncJob(ctx context.Context, localClient client.Clie
 				return true, nil
 			})
 		}
-		remoteFinished := false
-		for _, c := range remoteJob.Status.Conditions {
-			if (c.Type == batchv1.JobComplete || c.Type == batchv1.JobFailed) && c.Status == corev1.ConditionTrue {
-				remoteFinished = true
-				break
-			}
+
+		_, _, updated := fromObject(&remoteJob).Finished(ctx)
+		if remoteJob.Status.Active != localJob.Status.Active {
+			updated = true
 		}
-		if remoteFinished {
+		if updated {
```

Comment:
Actually, this change isn’t needed. We don’t copy the status when MultiKueueBatchJobWithManagedBy=false, so there’s no need to wait for Active=0.